package.json

```
@@ -1,39 +1,18 @@
 {
   "name": "use-timeout",
-  "version": "0.1.0",
-  "private": true,
+  "version": "1.0.0",
+  "main": "dist/index.js",
+  "types": "dist/index.d.ts",
   "dependencies": {
```

```suggestion
  "peerDependencies": {
```

React should be peer dep (unbundled).

cf. https://github.com/beizhedenglong/react-hooks-lib/blob/201fa36d1171c699c0f50a1bc210fb4b416cabc4/package.json#L79-L81